anoncreds/src/services/verifier.rs

```diff
@@ -337,7 +344,7 @@ fn compare_timestamps_from_proof_and_request(
                 received_self_attested_attrs
                     .get(referent)
                     .map(|_| ())
-                    .ok_or_else(|| err_msg!("Missing referent"))
+                    .ok_or_else(|| err_msg!("Referent validation failed: {}", referent))
```

Comment:
```suggestion
                    .ok_or_else(|| err_msg!("Referent validation failed: {referent}"))
```

I believe clippy complains about this now.